pkg/kubelet/volumemanager/populator/desired_state_of_world_populator_test.go

```diff
@@ -602,7 +609,8 @@ func TestEphemeralVolumeOwnerCheck(t *testing.T) {
 
 	podName := util.GetUniquePodName(pod)
 
-	dswp.findAndAddNewPods()
+	_, ctx := ktesting.NewTestContext(t)
```

Comment:
Hi @pohly , Can you take a look? `NewTestContext(t)` and `Init(t)`, which is preferred?

Reply:
`Init` is simpler and thus preferred.

Reply:
Just beware that it cannot be used in staging (yet) - I first need to move k/k/test/utils/ktesting into a staging repo.

Reply:
@carlory @pohly Thank you for your comments.

> I first need to move k/k/test/utils/ktesting into a staging repo.

Regarding this, is there anything I can help with, such as creating issues for repository creation at https://github.com/kubernetes/org?

Reply:
I first need to polish it a bit more and gather more experience with it before proposing it for staging.

Reply:
I changed to use `Init(t)` instead of `NewTestContext(t)`.
https://github.com/kubernetes/kubernetes/pull/124439/commits/aa67f41696d07aa5f333c6f2b7359548e074f267